examples/jobs/docker/hello_docker.aurora

```diff
@@ -26,10 +26,11 @@ hello_world_docker = Task(
   processes = [hello_world_proc],
   resources = Resources(cpu = 1, ram = 1*MB, disk=8*MB))
 
+p = [Parameter(name="label", value="instance${instance}")]
```

Comment:
I don't think the "${instance}" thing works. Can we revert all the changes in this file?